src/components/results/securityanalysis/security-analysis-result-tab.tsx

```diff
@@ -102,8 +102,25 @@ export const SecurityAnalysisResultTab: FunctionComponent<
         sortWay: SORT_WAYS.asc,
     });
 
+    const resultType = useMemo(() => {
+        if (tabIndex === N_RESULTS_TAB_INDEX) {
+            return RESULT_TYPE.N;
+        }
+
+        if (nmkType === NMK_TYPE.CONSTRAINTS_FROM_CONTINGENCIES) {
```

Comment:
```suggestion
        else if (nmkType === NMK_TYPE.CONSTRAINTS_FROM_CONTINGENCIES) {
```
To be coherent with the else above

Reply:
fixed